### ISSUE TITLE
Can't Replace Kazotsky Kick Sound

### ISSUE THREAD

**Issue by Vonaducci** [Batfoxkid/TF2-Custom-Taunts#16]
I'm trying to add a taunt that replaces the Kazotsky Kick, but whenever I taunt in game, it plays my custom sound and the original Kazotsky Kick sound. Here is the code for my taunt from customtaunts.cfg:

```
"Caramelldansen"
	{
		"models/player/scout.mdl"
		{
			"model"		"models/custom/taunts/caramell/scout.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/soldier.mdl"
		{
			"model"		"models/custom/taunts/caramell/soldier.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/pyro.mdl"
		{
			"model"		"models/custom/taunts/caramell/pyro.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/demo.mdl"
		{
			"model"		"models/custom/taunts/caramell/demo.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/heavy.mdl"
		{
			"model"		"models/custom/taunts/caramell/heavy.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/engineer.mdl"
		{
			"model"		"models/custom/taunts/caramell/engineer.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/medic.mdl"
		{
			"model"		"models/custom/taunts/caramell/medic.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/sniper.mdl"
		{
			"model"		"models/custom/taunts/caramell/sniper.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
		
		"models/player/spy.mdl"
		{
			"model"		"models/custom/taunts/caramell/spy.mdl"
			"index"		"1157"
			"duration"  "46"
			"sound"		"customtaunts/caramell/cossack_sandvich.mp3"
			
			"music/cossack_sandvich.wav"
			{
			}
		}
	}

```
I tried to rename "music/cossack_sandvich.wav" to "music/cossack_sandvich_loop.wav", but that didn't work either. How would I fix the taunt so that the original cossack_sandvich.wav doesn't play over my custom taunt sound? 

Lastly, despite using the kazotsky kick taunt index, the player cannot move when using this taunt. I saw in another post someone said to try adding "{ event AE_TAUNT_ENABLE_MOVE 0 "" } in your qc", but I have no idea what a qc is.

**Comment by codingh2** [comment]
> Lastly, despite using the kazotsky kick taunt index, the player cannot move when using this taunt. I saw in another post someone said to try adding "{ event AE_TAUNT_ENABLE_MOVE 0 "" } in your qc", but I have no idea what a qc is.

https://developer.valvesoftware.com/wiki/QC

You may need to recompile the model with that option. Don't worry, qc is a text file and you don't need to edit the model itself. 

You can use something like https://developer.valvesoftware.com/wiki/Mdldecompiler to decompile the model.

**Comment by Vonaducci** [comment]
> > Lastly, despite using the kazotsky kick taunt index, the player cannot move when using this taunt. I saw in another post someone said to try adding "{ event AE_TAUNT_ENABLE_MOVE 0 "" } in your qc", but I have no idea what a qc is.
> 
> https://developer.valvesoftware.com/wiki/QC
> 
> You may need to recompile the model with that option. Don't worry, qc is a text file and you don't need to edit the model itself.
> 
> You can use something like https://developer.valvesoftware.com/wiki/Mdldecompiler to decompile the model.

Okay, so I decompiled my scout.mdl with crowbar, found the .qc file and tried my best to guess where "{ event AE_TAUNT_ENABLE_MOVE 0 "" }" is supposed to go. I recompiled the file until crowbar stopped giving me errors regarding where I was putting "{ event AE_TAUNT_ENABLE_MOVE 0 "" }", but when I added these newly compiled files to my server, nothing changed. I still cannot walk using my taunt.

Here is my qc before editing:
https://github.com/Vonaducci/scouttaunt/blob/main/scout.qc

Here is my qc after editing:
https://github.com/Vonaducci/scouttaunt/blob/main/scout.qc_after_edit

I have no clue if I put "{ event AE_TAUNT_ENABLE_MOVE 0 "" }" on the correct line in the file (I put it on line 791), and looking up where to put it yields nothing.

Have I properly placed this line of code within my .qc and if not, where should I have put it?
